### DISCUSSION TITLE
neovimでモダンなRuby開発環境を構築する記事を書きたい

### DISCUSSION THREAD

**Discussion by joker1007** [rubima/magazine.rubyist.net#404] [記事を書きたい]
Steep、RBS、Typeprof、ruby-lsp、rubyfmt、debug.gemを使ったモダンなRuby開発環境をneovimで構築するガイドと、最新の開発体験を紹介する。

**Comment by hsbt** [comment]
ありがとうございます! 4月下旬の号はもう原稿締め切りになったので次の号でお願いします!

**Comment by takkanm** [comment]
ありがとうございますい！締め切りは過ぎましたが、今過ぐ記事を書いて PR だしていただいてもかまいません！なぜなら私は最近、`brew install nvim` したばかりだからです。

**Comment by kakutani** [comment]
#429 でお願いします!

**Comment by kakutani** [comment]
#430 こっちのほうがよさそう 🙏
